app/javascript/vue/shared/content/attachments/open_locally_menu.vue

```diff
@@ -1,11 +1,15 @@
 <template>
   <div class="sn-open-locally-menu" @mouseenter="fetchLocalAppInfo">
     <div v-if="!canOpenLocally && (attachment.attributes.wopi && attachment.attributes.urls.edit_asset)">
-      <a :href="`${attachment.attributes.urls.edit_asset}`" target="_blank"
+      <a v-if="editWopiEnabled" :href="`${attachment.attributes.urls.edit_asset}`" target="_blank"
```

Comment:
would it work if you just add the btn-disabled class here using this:
`:class="{ editWopiEnabled: 'btn-disabled' }"`

Reply:
@artoscinote Not sure if we have styles defined for **btn-disable**. Using `:class="{ 'disabled': !editWopiEnabled }"` could be of help except for that there will be no pointer events on link and we want to show tooltip on hover for unsupported format